docs/usage/zcml.md

```diff
@@ -0,0 +1,65 @@
+# Registration
+
+The common way to register a new distribution is via zcml.
+
+First, you need to include `plone.distribution` as dependency of your package by adding the following snippet to your `configure.zcml`:
+
+```xml
+  <include package="plone.distribution" />
+```
+
+Then declare the distributions included in your package using the `plone:distribution` configuration:
+
+```xml
+  <plone:distribution
+      name="blog"
+      title="Personal Blog"
+      description="A Plone site already configured to host a personal Blog."
+      directory="distributions/blog"
+      />
+```
+
+## Options
+
+| attribute | description | default value | required |
+|--|--|--|--|
+| name | Identifier of the distribution |  | ✅ |
+| title | Friendly name of the distribution |  | ✅ |
+| description | An explanation of the distribution |  |  |
+| directory | Distribution directory | distributions/`<name>` |  |
+| pre_handler | Callable function to process answers before site creation |  |  |
+| handler | Callable function used by `plone.distribution` to handle site creation. |  |  |
+| post_handler | Callable function used to make adjustments after site is created  |  |  |
+
+## Examples
+
```

Comment:
All headings should have some content, even though it may be obvious.

What do you think of this? Did I guess right? Feel free to modify.

```suggestion

This section provides examples of this package being used in production.
They may serve as useful references for developing your own distribution.

```